packages/node/src/utils/ensureIsWrapped.ts

```diff
@@ -24,5 +24,10 @@ export function ensureIsWrapped(
         );
       }
     });
+
+    getCurrentScope().setContext('Instrumentation', {
+      isMissing: true,
+      framework: name,
```

Comment:
l: not sure if `framework` is the ideal word here, maybe `package` or something like this is more fitting? 🤔 but no strong feelings...

Also, I think it would be useful to also add `isCjs: isCjs()` there, this is information we often need for debugging :)

Reply:
+1 for `isCjs()`